pkg/metrics/metrics.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var (
+	keyName, _      = tag.NewKey("name")
+	keyFleetName, _ = tag.NewKey("fleet_name")
+	keyType, _      = tag.NewKey("type")
+
+	fleetsReplicasCountStats = stats.Int64("fleets/replicas_count", "The count of replicas per fleet", "1")
```

Comment:
I hesitated a lot between global scope and controller variable, in the end it's easier to deal with global scope but also cleaner to arrange and declare.

Couldn't find any example not using global scope for OpenCensus, are we ok with this ?

@markmandel @enocom

Reply:
I also like having all views/stats declaration in one file.

Reply:
Good idea, so you’re fine with the rest of this file ?

Reply:
I'm not seeing any issues here I don't think. Is this data mutated, or is it just a declaration? Especially if it's just a declaration, I think it makes perfect sense. (I think it's just a declaration, if I read it correctly?)

Reply:
Yes it’s declaration only